packages/grpc-js-core/README.md

```diff
@@ -18,5 +18,14 @@ npm install @grpc/grpc-js
  - TLS channel credentials
  - Call credentials (for auth)
  - Simple reconnection
+ - Channel API
 
 This library does not directly handle `.proto` files. To use `.proto` files with this library we recommend using the `@grpc/proto-loader` package.
+
+## Some Notes on API Guarantees
+
+The public API of this library follows semantic versioning, with some caveats:
```

Comment:
Maybe reflect some of that in the README of the native implementation ?

Reply:
Yeah, there's no urgency anyway. I was thinking about doing the same comment about underscore and private members that aren't supposed to be part of the public API.

We might also want to start tagging private members using underscores :)